src/System.Private.CoreLib/shared/System/Runtime/CompilerServices/AsyncMethodBuilder.cs

```diff
@@ -29,24 +29,24 @@ public struct AsyncVoidMethodBuilder
     {
         /// <summary>The synchronization context associated with this operation.</summary>
         private SynchronizationContext? _synchronizationContext;
-        /// <summary>The builder this void builder wraps.</summary>
-        private AsyncTaskMethodBuilder _builder; // mutable struct: must not be readonly
+        /// <summary>The lazily-initialized built task.</summary>
```

Comment:
Are all these changes to AsyncVoidMethodBuilder really necessary?  Devs that care about perf are using "async void"?

Reply:
The methods are there now, so its just swapping from forwarding to `AsyncTaskMethodBuilder` to `AsyncTaskMethodBuilderCore`